config/locales/en.yml

```diff
@@ -72,6 +72,10 @@ en:
       news_and_communications: News and communications
       statistics: Statistics
       worldwide: Worldwide
+    intervention:
+      title: Check the next steps for your limited company
+      description: You might be interested in this because you’ve been browsing guidance relevant to starting a limited company.
+      dismiss_html: <a class="govuk-link" href="?hide-intervention=true">Hide this suggestion</a> if it’s not relevant to you
```

Comment:
I'm wondering how this link will be handled - there's no explanation in the documentation.

Reply:
Added:

    The dismiss link will reload the page but the `hide-intervention` query string parameter will cause the
    backed not to show the intervention again. Some progressive enhancement will be added in later to avoid
    reloading the page if JavaScript is available.